src/data/multidim/multidimindex.h

```diff
@@ -115,8 +115,7 @@ class SubSliceIndex : public std::vector<SliceIndex>
 	{
 		typedef Text::SmartString S;
 		return "[ "
-		     + S::join<std::vector, SliceIndex>(*this,
-		         std::string(", "))
+		     + S::join(*this, ", ")
```

Comment:
This should be toJSON() as well.

Reply:
I'll change it in https://github.com/vizzuhq/vizzu-lib/pull/297 PR

Reply:
https://github.com/vizzuhq/vizzu-lib/pull/297/commits/30d727829e141581170bc03d38d1c5cdc352678c